kubernetes/spec/utils_spec.rb

```diff
@@ -86,18 +86,5 @@
         expect(Kubernetes.create_temp_file_with_base64content(content)).to eq(expected_path)
       end
     end
-
-    context 'when it is already called' do
```

Comment:
@DavidHuie We should maybe adjust this test so that it works with the given refactoring. 

If we remove it entirely we don't have a test that checks the caching functionality isn't removed.

Reply:
Good call. I re-wrote the test to use the newest bit of code.